emrun.py

```diff
@@ -38,10 +38,11 @@
 from operator import itemgetter
 
 if sys.version_info.major == 2:
+  from urllib import unquote
+
   import SocketServer as socketserver
   from BaseHTTPServer import HTTPServer
   from SimpleHTTPServer import SimpleHTTPRequestHandler
-  from urllib import unquote
   from urlparse import urlsplit
```

Comment:
what is the rule here, do you know? the new version seems... arbitrary at first glance.